packages/kokoas-client/src/pages/projSearch/sections/result/details/paymentDetails/unissuedInvoiceAlert/notifyAlertToCharwork/createMessage.ts

```diff
@@ -35,7 +36,8 @@ export const createMessage = async ({
     recContracts,
   });
 
-  const displayDate = paymentDate ? `${format(paymentDate, 'yyyy年MM月dd日')}に` : '';
+  const hasDisplayDate = getDisplayPaymentDate(paymentDate);
```

Comment:
[hasは普段booleanに付ける](https://tamcom.jp/articles/naming-of-bool-methods)ので、他のところ同じように、以下でいいです。

```suggestion
  const displayDate = getDisplayPaymentDate(paymentDate);
```

Reply:
ご指摘ありがとうございます。
[a2771e1](https://github.com/cocosumo/yumecoco-monorepo/pull/1250/commits/a2771e15e2d88e109ccee897128ca031ee3a1ff0)にて修正しましたので、ご確認をお願いいたします。